styles.css

```diff
@@ -0,0 +1,270 @@
+body {
+	background-color: #181818;
+	color: white;
+	margin: 0px;
+	font-family: 'Roboto', sans-serif;
+	display: block;
+}
+
+#navbar {
```

Comment:
For the navbar, the layout is breaking, as you can see in the screenshot. Please fix it.
![navissue](https://user-images.githubusercontent.com/33268927/169064353-b15c1049-fca2-4619-9a1c-bdb8f033ec90.png)